porcupine/plugins/pastebin.py

```diff
@@ -246,7 +247,66 @@ def pasting_done_callback(
         )
 
 
+def ask_are_you_sure(filename: str | None, paste_class: type[Paste]) -> bool:
+    window = tkinter.Toplevel()
+    window.title(f"Pastebin {filename}")
+
+    content = ttk.Frame(window, name="content", padding=10)
+    content.pack(fill="both", expand=True)
+    content.columnconfigure(0, weight=1)
+
+    label1 = ttk.Label(
+        content,
+        name="label1",
+        text=f"Do you want to send the content of {filename} to {paste_class.name}?",
```

Comment:
This text can be misleading, it says, `the content of {filename}` even if I just try to pastebin the selection.